meta-resin-common/recipes-bsp/u-boot/patches/env_resin.h

```diff
@@ -102,6 +113,8 @@
                       "fi;" \
               "fi;\0" \
        \
+       "set_os_cmdline=" \
+               "setenv os_cmdline ${base_os_cmdline} ${bootparam_flasher};\0" \
```

Comment:
For non-flasher devices, `bootparam_flasher` isn't defined. Does u-boot not complain?

Reply:
@ZubairLK No. It ends up empty. You can't define an empty variable in uboot.

Reply:
:+1: